doc/source/user_guide/groupby.rst

```diff
@@ -1268,6 +1268,11 @@ automatically excluded. In other words, there will never be an "NA group" or
 generally discarding the NA group anyway (and supporting it was an
 implementation headache).
 
+.. note::
+   If you need to include NaN or NaT values in your grouping, you can workaround the automatic exclusion by replacing
+   the NaN / NaT values with a placeholder string. For example, you can use ``df.fillna("default", inplace=True)`` to
```

Comment:
Thanks for the PR! 

1.  Could you demonstrate this by including a `.. ipython:: python` example (like above)
2. In the example, could you exclude `inplace=True`? We are trying to discourage that keyword generally

Reply:
Can do on both!

Do you think it makes sense to pull it out of a note and just include it in line with the rest of the text if we're including an ipython?

Reply:
Yeah sure that sounds reasonable